src/auth/cephx/CephxKeyServer.cc

```diff
@@ -133,6 +133,7 @@ bool KeyServerData::get_caps(CephContext *cct, const EntityName& name,
 
 KeyServer::KeyServer(CephContext *cct_, KeyRing *extra_secrets)
   : cct(cct_),
+    temp_data(extra_secrets),
```

Comment:
s/temp_data/pending_data/g

and shall we reset the `pending_data` once the incremental is applied.